HCP docs update for IBM Power (#92359)
openshift/openshift-docs: modules/hcp-scale-the-nodepool.adoc

Reviewer: These steps mix a procedure and a few different verification steps. I am having a hard time understanding this one.

NAME                                   CLUSTER         APPROVED   ROLE          STAGE
4dac1ab2-7dd5-4894-a220-6a3473b67ee6   hypercluster1   true       auto-assign  
d9198891-39f4-4930-a679-65fb142b108b                   true       auto-assign  
da503cf1-a347-44f2-875c-4960ddb04091   hypercluster1   true       auto-assign
----
+
[source,yaml]
----
oc -n ${HOSTED_CONTROL_PLANE_NAMESPACE} get agent -o jsonpath='{range .items[*]}BMH: {@.metadata.labels.agent-install\.openshift\.io/bmh} Agent: {@.metadata.name} State: {@.status.debugInfo.state}{"\n"}{end}'
BMH: ocp-worker-2 Agent: 4dac1ab2-7dd5-4894-a220-6a3473b67ee6 State: binding
BMH: ocp-worker-0 Agent: d9198891-39f4-4930-a679-65fb142b108b State: known-unbound
BMH: ocp-worker-1 Agent: da503cf1-a347-44f2-875c-4960ddb04091 State: insufficient
----
+ 
. Once the agents reach the `added-to-existing-cluster` state, check the OpenShift nodes after a few moments:
+
[source,yaml]
----
oc --kubeconfig ${HOSTED_CLUSTER_NAME}.kubeconfig get nodes
NAME           STATUS   ROLES    AGE     VERSION
ocp-worker-1   Ready    worker   5m41s   v1.24.0+3882f8f
ocp-worker-2   Ready    worker   6m3s    v1.24.0+3882f8f
----
+

> . Adding workloads to the nodes can reconcile some cluster operators. The following command displays that two machines are created after scaling up the node pool:
> +
> [source,yaml]
> ----
> oc -n ${HOSTED_CONTROL_PLANE_NAMESPACE} get machines
> NAME                            CLUSTER               NODENAME       PROVIDERID                                     PHASE     AGE   VERSION
> hypercluster1-c96b6f675-m5vch   hypercluster1-b2qhl   ocp-worker-1   agent://da503cf1-a347-44f2-875c-4960ddb04091   Running   15m   4.11.5
> hypercluster1-c96b6f675-tl42p   hypercluster1-b2qhl   ocp-worker-2   agent://4dac1ab2-7dd5-4894-a220-6a3473b67ee6   Running   15m   4.11.5
> ----

Reviewer: ```suggestion
. Adding workloads to the nodes might reconcile some cluster Operators. The following command displays that two machines are created after scaling up the node pool:
+
[source,terminal]
----
$ oc -n ${HOSTED_CONTROL_PLANE_NAMESPACE} get machines
----
+
.Example output
----
NAME                            CLUSTER               NODENAME       PROVIDERID                                     PHASE     AGE   VERSION
hypercluster1-c96b6f675-m5vch   hypercluster1-b2qhl   ocp-worker-1   agent://da503cf1-a347-44f2-875c-4960ddb04091   Running   15m   4.11.5
hypercluster1-c96b6f675-tl42p   hypercluster1-b2qhl   ocp-worker-2   agent://4dac1ab2-7dd5-4894-a220-6a3473b67ee6   Running   15m   4.11.5
----
```

+

> . Access the hostcluster by using the `kubeconfig`:
> +
> [source,yaml]
> ----
> hypershift create kubeconfig –namespace <hosted_control_plane_namespace> –name example >> example.kubeconfig
> ----

Reviewer: What does this mean?
```suggestion
. To access the host cluster by using the `kubeconfig`, run the following command:
+
[source,terminal]
----
$ hypershift create kubeconfig –namespace <hosted_control_plane_namespace> –name example >> example.kubeconfig
----
```

// Module included in the following assemblies:
//
// * hosted_control_planes/hcp-deploy/hcp-deploy-ibm-power.adoc

:_mod-docs-content-type: PROCEDURE
[id="hcp-scale-the-nodepool_{context}"]
== Scaling the node pool

After your agents are approved, you can scale the node pools. The `agentLabelSelector` value that is configured in the node pool ensures that only matching agents are added to the cluster. This also helps scale down the node pool. To remove specific architecture nodes from the cluster, scale down the corresponding node pool.

.Procedure

> . Scale the node pool by running the following command:

Reviewer 2: This seems like a one-step procedure, so it should be an asterisk:

```suggestion
* Scale the node pool by running the following command:
```

+
[source,terminal]
----
$ oc -n <clusters_namespace> scale nodepool <nodepool_name> --replicas 2
----
+
[NOTE]
====
The Cluster API agent provider picks two agents randomly to assign to the hosted cluster. These agents pass through different states and then join the hosted cluster as {product-title} nodes. The various agent states are `binding`, `discovering`, `insufficient`, `installing`, `installing-in-progress`, and `added-to-existing-cluster`.
====

[id="hcp-scale-nodepool-verifying_{context}"]

> === Verifying the scaling of the node pools

Reviewer 2: This is nested pretty deep in the content. Also, because the content above this counts as "a procedure," this content would technically need to be in a different module, as we only allow one procedure per module. This either needs split out, or it needs to be formatted as a more traditional '.Verification' section we do in procedures (in this scenario, you'd remove lines 25-26 and replace them with .Verification):

```suggestion
.Verification
```
